models/examples/netbox/netbox.yml

```diff
@@ -242,7 +242,7 @@ nodes:
     description: "A Circuit represent a single physical link between two locations"
     label: "Circuit"
     icon: "mdi:transit-connection-variant"
-    default_filter: circuit_id__value
+    human_friendly_id: ["name__value"]
```

Comment:
Same as the above, it should be **circuit_id__value**

Reply:
@BaptisteGi I've changed them all now.